nemo_skills/dataset/swe-rebench/dump_images.py

```
@@ -0,0 +1,128 @@
+
+    # Check if SIF file already exists
+    if os.path.exists(sif_path):
+        print(f"✓ {container_name} -> {sif_path} (already exists)")
+        return True, container_name, sif_path
+
+    try:
+        # Use apptainer to build SIF from Docker container
+        cmd = ["apptainer", "build", sif_path, f"docker://{container_name}"]
+
+        print(f"Converting {container_name}...")
+        result = subprocess.run(cmd, capture_output=True, text=True)
```

Ruff (0.14.10)

`subprocess` call: check for execution of untrusted input

(S603)